packages/ember-glimmer/tests/integration/helpers/inline-if-test.js

```diff
@@ -0,0 +1,150 @@
+import { moduleFor } from '../../utils/test-case';
+import { set } from 'ember-metal/property_set';
+import EmberObject from 'ember-runtime/system/object';
+import jQuery from 'ember-views/system/jquery';
+
+import {
+  BASIC_TRUTHY_TESTS,
+  BASIC_FALSY_TESTS,
+  SharedHelperConditionalsTest
+} from '../../utils/shared-conditional-tests';
+
+moduleFor('Helpers test: inline {{if}}', class extends SharedHelperConditionalsTest {
+
+  templateFor({ cond, truthy, falsy }) {
+    return `{{if ${cond} ${truthy} ${falsy}}}`;
+  }
+
+  ['@test it allows static arguments']() {
+    this.render(`{{if condition 'yes' 'no'}}`, { condition: true });
+
+    this.assertText('yes');
+
+    this.runTask(() => this.rerender());
+
+    this.assertText('yes');
+
+    this.runTask(() => set(this.context, 'condition', false));
+
+    this.assertText('no');
+  }
```

Comment:
A more general point (even though I don't think we need this test): the general pattern we have been following is:

1. Initial render
2. No-op re-render
3. Change (via mutation if applicable)
4. Repeat (3) as needed (sometimes with replacement, or other values, sometimes grouped, ...)
5. Reset (via replacement if applicable)

In this case, you would change render it with `true` initially, then `rerender` with no change, then change it to `false`, and finally change it back to `true` (replacement vs mutation doesn't make a difference here).

See https://github.com/emberjs/ember.js/pull/12927#discussion_r52392940 for more details